File: server/src/com/broll/gainea/server/core/actions/NoActionContext.java
package com.broll.gainea.server.core.actions;

import com.broll.gainea.net.NT_Action;

public class NoActionContext<T extends NT_Action> extends ActionContext<T> {

    public NoActionContext(T action) {
        super(action);
    }

    public NoActionContext(T action, CustomReactionHandler<NoActionContext> handler) {
        this(action);
        this.setCustomHandler(handler);
    }

}

File: server/src/main/com/broll/gainea/server/core/actions/NoActionContext.kt
package com.broll.gainea.server.core.actions

import com.broll.gainea.net.NT_Action

class NoActionContext<T : NT_Action?>(action: T) : ActionContext<T>(action) {
    constructor(action: T, handler: CustomReactionHandler<NoActionContext<*>?>?) : this(action) {
        customHandler = handler
    }
}
